builder_test.go

```diff
@@ -775,9 +776,6 @@ func TestBuilder(t *testing.T) {
 				Image: "centos:7",
 				Shell: []string{"/bin/bash", "-xc"},
 			},
-			ErrFn: func(err error) bool {
-				return err != nil && strings.Contains(err.Error(), "multiple FROM statements are not supported")
-			},
```

Comment:
I'm not sure why this is getting removed?  More tea for me?

Reply:
The Dockerfile the test uses doesn't have multiple FROM statements in it, so the function was never called.  Under the new paradigm, having a function that decides whether or not to ignore an error, but not having an error to pass to it, would cause the test to fail.

Reply:
Thanks, thought this test had multiple FROM calls.  SGTM.